"""
Zbog malo cudnog nacina na koji
radi zadatak prilikom pokretanja programa
u main.py dolazi do nastanka "Circular Dependecies",
tj da bi func.py radio kako treba potreban mu je main.py
a main.py je potreban func.py tako da nastaje petlja,
python ovo prepoznaje i izbacuje gresku, moguce je popraviti
ali mi se nije dalo :)
"""



